scripts/ci/travis-script.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Script that runs in every Travis CI container. The script is responsible for delegating
```

Comment:
Yeah. I think there is some confusion about the `material2-build-tools` and those scripts right now. We should have a chat about the things we want to externalize and how we are going to approach this.